src/pages/public_registration/ActionPanel.tsx

```diff
@@ -33,13 +88,17 @@ export const ActionPanel = ({
         sx={{ bgcolor: 'primary.main', px: '0.5rem' }}
         textColor="inherit"
         TabIndicatorProps={{ style: { backgroundColor: 'white', height: '0.4rem' } }}>
+        {canSeeTasksPanel && (
+          <Tab
```

Comment:
Ideelt sett skulle vi kanskje ikke rendera `<Tabs>` her i det hele tatt når det er bare en fane som er gyldig, men ser ut som det går fint etter testing av a11y, så tenker jeg dropper den strafferunden nå.
En annen ting vi kunne vurdert er å skrive om dette til å bruke den nye `<TabsList>` som er i `@mui/lab` og vi bruker for NVI-kandidater, men syns det er feil å prioritere tid på det nå